### ISSUE TITLE
您好请问一下我在做训练时acc一直为0 使用训练的数据ocr不能识别出来这是什么原因导致的？谢谢了！

### ISSUE THREAD

**Issue by yangjian9527** [PaddlePaddle/PaddleOCR#2605]
这是我的配置文件，这应该是使用的这个模型往上继续加数据吧，难道是从头训练的么。
如果是我想再官方模型的基础上加数据应该怎么操作，谢谢解答。
![image](https://user-images.githubusercontent.com/33308470/115840691-86befd00-a44e-11eb-9242-99701298b0ca.png)

**Comment by yangjian9527** [comment]
Global:
  use_gpu: false
  epoch_num: 1000
  log_smooth_window: 100
  print_batch_step: 1
  save_model_dir: ./output/rec_chinese_common_v2.0
  save_epoch_step: 3
  # evaluation is run every 5000 iterations after the 4000th iteration
  eval_batch_step: [0, 2000]
  cal_metric_during_train: True
  pretrained_model: PPOCRLabel/pretrain_models/ch_ppocr_mobile_v2.0_rec_pre/best_accuracy
  checkpoints:
  save_inference_dir: ch_ppocr_server_v2.0_rec_infer
  use_visualdl: False
  infer_img: doc/imgs_words/ch/word_2.jpg
  # for data or label process
  character_dict_path: ./ppocr/utils/ppocr_keys_v1.txt
  character_type: ch
  max_text_length: 25
  infer_mode: False
  use_space_char: True


Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0.00004

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: ResNet
    layers: 34
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 256
  Head:
    name: CTCHead
    fc_decay: 0.00004

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data
    label_file_list: ["./train_data/ic15_data/train/rec_gt_train.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - RecAug: 
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 32
    drop_last: True
    num_workers: 8

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data
    label_file_list: ["./train_data/ic15_data/test/rec_gt_test.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 32
    num_workers: 8

**Comment by littletomatodonkey** [comment]
配置文件和预训练模型保持一致吧，你这没对应上

**Comment by yangjian9527** [comment]
> 配置文件和预训练模型保持一致吧，你这没对应上

common的配置文件需要的是Server的模型么

**Comment by MissPenguin** [comment]
> > 配置文件和预训练模型保持一致吧，你这没对应上
> 
> common的配置文件需要的是Server的模型么

对的
